frontend/src/components/user-management/student/AddClassroomConfirmationMessage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
i believe there should be a toast and not this confirmation message

<img width="738" alt="image" src="https://user-images.githubusercontent.com/69697180/230817732-ca301258-833b-4d84-a096-cc6c3a517996.png">